migrations/20190428113359-create-articlecomment.js

```diff
@@ -0,0 +1,36 @@
+
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('articlecomment', {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    userid: {
```

Comment:
thanks, @jnkindi, I looked both the model and the migration file, The two fields **userid** and **articleid** specified, makes sense that the comment belongs to a user and to a specific article, so far, this means that both **userid** and **articleid** are **foreign keys** to **comment table**. yet i wasn't able to se the association for the 2 specified.